pkgs/tools/admin/lxd/default.nix

```diff
@@ -98,6 +98,7 @@ buildGoModule rec {
 
   passthru.tests.lxd = nixosTests.lxd;
   passthru.tests.lxd-nftables = nixosTests.lxd-nftables;
+  passthru.updateScript = ./update.sh;
```

Comment:
I think we can use `gitUpdater`
```suggestion
  passthru.updateScript = gitUpdater {
    url = "https://github.com/lxc/lxd.git";
    rev-prefix = "lxd-";
  };
```

Reply:
Yes, this does work. How can I even know such an updater exists? It's not in the manual.

Reply:
Unfortunately, I don't have an answer. I just saw it being used by other packages.

Reply:
Well thanks for sharing. Better than my shell script for sure. :)